tests/integration/test_charm.py

```diff
@@ -332,6 +337,215 @@ async def test_remove_postgresql_metastore_relation(ops_test: OpsTest, test_pod,
+    print(process.stderr.decode())
+    logger.info(f"JDBC endpoint test returned with status {process.returncode}")
+    assert process.returncode == 0
+
+
+@pytest.mark.abort_on_fail
+async def test_set_password_action(ops_test: OpsTest):
+    """Test set-password action."""
+    logger.info("Running action 'get-password' on kyuubi-k8s unit...")
+    kyuubi_unit = ops_test.model.applications[APP_NAME].units[0]
+    action = await kyuubi_unit.run_action(
+        action_name="get-password",
+    )
+    result = await action.wait()
+    old_password = result.results.get("password")
+
+    logger.info("Running action 'set-password' on kyuubi-k8s unit...")
+    password_to_set = str(uuid.uuid4())
+    action = await kyuubi_unit.run_action(action_name="set-password", password=password_to_set)
+    result = await action.wait()
+    assert result.results.get("password") == password_to_set
+
+    logger.info("Running action 'get-password' on kyuubi-k8s unit...")
+    action = await kyuubi_unit.run_action(
+        action_name="get-password",
+    )
+    result = await action.wait()
+    new_password = result.results.get("password")
+
+    assert new_password != old_password
+    assert new_password == password_to_set
+
+
+@pytest.mark.abort_on_fail
+async def test_remove_authentication(ops_test: OpsTest, test_pod, charm_versions):
+    """Test the JDBC connection when authentication is disabled."""
+    logger.info("Removing relation between postgresql-k8s and kyuubi-k8s over auth-db endpoint...")
+    await ops_test.model.applications[APP_NAME].remove_relation(
+        f"{APP_NAME}:auth-db", f"{charm_versions.postgres.application_name}:database"
+    )
+
+    logger.info("Waiting for postgresql-k8s and kyuubi-k8s apps to be idle and active...")
+    await ops_test.model.wait_for_idle(
+        apps=[APP_NAME, charm_versions.postgres.application_name], timeout=1000, status="active"
+    )
+    logger.info("Running action 'get-jdbc-endpoint' on kyuubi-k8s unit...")
+    kyuubi_unit = ops_test.model.applications[APP_NAME].units[0]
+    action = await kyuubi_unit.run_action(
+        action_name="get-jdbc-endpoint",
+    )
+    result = await action.wait()
+
+    jdbc_endpoint = result.results.get("endpoint")
+    logger.info(f"JDBC endpoint: {jdbc_endpoint}")
+
+    logger.info("Testing JDBC endpoint by connecting with beeline with no credentials ...")
+    process = subprocess.run(
+        [
+            "./tests/integration/test_jdbc_endpoint.sh",
+            test_pod,
+            jdbc_endpoint,
+            "db_444",
+            "table_444",
+        ],
+        capture_output=True,
+    )
+    print("========== test_jdbc_endpoint.sh STDOUT =================")
+    print(process.stdout.decode())
+    print("========== test_jdbc_endpoint.sh STDERR =================")
+    print(process.stderr.decode())
+    logger.info(f"JDBC endpoint test returned with status {process.returncode}")
+    assert process.returncode == 0
```

Comment:
Here we need to check that login will failed with the old credentials

Reply:
When the authentication in Kyuubi is removed, Kyuubi accepts all connections and doesn't check the credentials at all. That's why, the login would not fail with the old credentials as authentication is removed.